src/utils.ts

```diff
@@ -47,6 +59,28 @@ function hasDartExecutable(pathToTest: string): boolean {
 	return false; // Didn't find it, so must be an invalid path.
 }
 
+export function findFlutterSdk(): string {
+	let paths = (<string>process.env.PATH).split(path.delimiter);
+
+	let flutterPath = paths.find(hasFlutterExecutable);
+	if (!flutterPath)
+		return null;
+	
+	let realFlutterPath = fs.realpathSync(path.join(flutterPath, flutterExecutableName));
+
+	return path.join(path.dirname(realFlutterPath), "..");
+}
+
+function hasFlutterExecutable(pathToTest: string): boolean{
```

Comment:
Very minor but since this whole function is a copy/paste with just the filename changed, I think it'd be better to rename `hasExecutable` and just pass the full path/filename in.